docs/update-locking.md

```diff
@@ -16,9 +16,33 @@ assumption that updates are locked (see [issue #20](https://github.com/resin-io/
 
 In supervisors >= v4.0.0 and any OS that is not Resin OS 1.X, the old lock location is completely ignored.
 
-### Taking the lock
+### Creating the lockfile
 
-Using the above-mentioned library, the lock can be acquired like in this CoffeeScript example:
+There are many different tools and libraries to provide proper lockfile functionality. Here are a few examples, and you can search online for others in other programming languages not mentioned here explicitly.
+
+__Note:__ Just creating the lockfile (for example by `touch /tmp/resin/resin-updates.lock`) is not adequate to prevent updates. That file way won't have the exclusive access flag set, and thus does not provide reliable locking.
```

Comment:
Nice explanation!

NIT
- `for example by touch` --> `for example by using touch`
- `That file way won't` --> `A file created in this way won't`

Reply:
Updated